### ISSUE TITLE
"Check build quality" checks tasks that were not executed due to custom condition

### ISSUE THREAD

**Issue by utna666** [MicrosoftPremier/VstsExtensions#35]
If a task did not execute due to a custom condition, "check build quality" waits for log information of that task and fails after the timeout. Is there any way to bypass this behavior?
![2019-06-03_11h00_23](https://user-images.githubusercontent.com/5910372/58790163-16a3f580-85f0-11e9-8909-534f907590d5.png)

**Comment by ReneSchumacher** [comment]
Thanks for reporting this issue. We can reproduce it and it clearly is a bug that will be fixed with the next version. I'll update this issue once the fixed version is available.

Thanks
René

**Comment by ReneSchumacher** [comment]
Hi again,

thanks for you patience! There was a delay due to us waiting on a backend change in Azure DevOps. The fix is now ready for production and the new task version should be available by tomorrow at the latest. I'm closing the issue now, but please feel free to open another one if you keep hitting this or any other issue.

Thanks,
René
